### PR TITLE
4.0.x

### PR DESCRIPTION
**Pull request by ruudboon**
* Type: bug fix | new code quality
* Link to issue: #14609 

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:
- Changed travis to use php 7.4.0 release
- Fixed bug in GD

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: php
 php:
-  - '7.4snapshot'
+  - '7.4'
```
Comment (sergeyklay) [inline .travis.yml]: Could you please remove 7.4snapshot from the allow_failures matrix

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (ruudboon) [reply .travis.yml]: I think we can remove the line completely. Right? @sergeyklay

Comment (sergeyklay) [reply .travis.yml]: Also please check `if [ "$(php-config --vernum)" -ge "70400" ]` condition in this file. Do we still need it?

Comment (ruudboon) [reply .travis.yml]: @sergeyklay Upgrade `Development Zephir version` to 7.4 as well?

**Comment by niden** [comment]
@ruudboon while you are here can you add this to the `phpcs.xml`

`<arg value="p">`

I like seeing the dots when the inspection runs. Gives you assurance :)

**Review by niden — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (sergeyklay) [reply .travis.yml]: Yes, I think so

**Review by niden — APPROVED** [review]

**Review by sergeyklay — APPROVED** [review]

**Pull request merged**